CHANGELOG.md

```diff
@@ -11,6 +11,7 @@ Note that this project **does not** adhere to [Semantic Versioning](http://semve
 
 ### Added
 
+- We implemented an IdBasedParserFetcher for zbmath.
```

Comment:
The changelog is mostly for our users. Could you thus provide a more "understandable" explanation. Thanks!

Reply:
I changed the changelog.